scripts/utils.js

```diff
@@ -240,46 +241,77 @@ function buildAuthorHeader(mainEl) {
   div.prepend(authorHeader);
 }
 
-async function buildArticleHeader(el) {
+/**
+ * * @param {HTMLElement} element
+ * * @param {string} targetTag, like 'ul' or 'div'
+ * result: return the new element with inner content of the element, desired tag and css class
+ */
+export function changeHTMLTag(el, targetTag, properties = {}) {
+  const newEl = document.createElement(targetTag);
+  const innerContent = el.innerHTML;
+
+  Object.keys(properties).forEach((key) => {
+    newEl.setAttribute(key, properties[key]);
+  });
+
+  newEl.innerHTML = innerContent;
```

Comment:
nit, `innerContent` temp var not needed
```suggestion
  Object.keys(properties).forEach((key) => {
    newEl.setAttribute(key, properties[key]);
  });

  newEl.innerHTML = el.innerHTML;
```

Reply:
updated, thanks